x-team/auto-ui
# Render Note content as raw text if it's not parseable by `markdown-to-jsx`

**Release Type:** *Bug Fix*

Fixes https://x-team-internal.atlassian.net/browse/XP-2275

## Description

Due to bugs in the library `markdown-to-jsx`, there are some texts that cause it to fail when trying to parse it as markdown.

This PR aims to fallback to show the raw text when that happens instead of letting the component throw an exception.

Created an issue in the repo for `markdown-to-jsx`: https://github.com/probablyup/markdown-to-jsx/issues/226

## Checklist


## Related PRs

branch | PR
------ | ------
XP-2275-bug-posting-note | [Auto](https://github.com/x-team/auto/pull/1643)

## Steps to Test or Reproduce

See the example file in this pull request.

One of the cases where it fails is with this text: `[one] [two]`

## Impacted Areas in Application

The `Note` component

## Screenshots

**Before**

![note before](https://user-images.githubusercontent.com/7514993/48632536-84ac0680-e997-11e8-9ba2-6cdb052778f1.PNG)

**After**

![note after](https://user-images.githubusercontent.com/7514993/48632544-8970ba80-e997-11e8-900f-e910d8b6ee79.PNG)
